### ISSUE TITLE
assigning voters with no verified voters, then got to voting, nomethoderror

### ISSUE THREAD

**Issue by ywwg** [FireflyArtsCollective/ffagc#43]
Showing /home/owen/src/github/ffagc/app/views/votes/index.html.erb where line #57 raised:

undefined method `score_t' for nil:NilClass

===================
```
 <select id="t[<%= gs.id %>]" name="t[<%= gs.id %>]" onchange="ajax_submit('#save<%= gs.id %>', 'vote_form')">
            <option></option>
            <option <%= @votes[gs.id].score_t == 0 ? "selected" : "" %>>0</option>
            <option <%= @votes[gs.id].score_t == 1 ? "selected" : "" %>>1</option>
            <option <%= @votes[gs.id].score_t == 2 ? "selected" : "" %>>2</option>
          </select>
```

**Comment by ywwg** [comment]
was logged in as artist and admin, not voter (hadn't verified yet)

**Comment by ywwg** [comment]
still doesn't work even after creating a grant and assigning again.

**Comment by ywwg** [comment]
I think this was also fixed in b444b0351ca8bd6ceaf443dc1edffb22446ee73f
